run-snyk-tests.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+# Constants to add colour to text

```

Comment:
Can you add a comment to the top of this file explaining what it is for?

Reply:
Added.